src/api/ReducerClient.js

```diff
@@ -0,0 +1,39 @@
+import BaseClient from "api/BaseClient";
+import { registerHistoryStackPushEvent } from "hooks/PreviousQuotesHistory";
+
+class ReducerClient extends BaseClient {
```

Comment:
This is an action, not a reducer ?

Reply:
The `PreviousQuotes` page uses a Reducer to manage state, and this API Client needs to updated the state using the reducer's `dispatch` function with the AJAX response. What's an `action` anyway?